docs/development/decision-records/2023-04-03_renaming_branches/README.md

```diff
@@ -0,0 +1,61 @@
+# Renaming Git branches to comply with TractusX standards
+
+## Decision
+
+TractusX-EDC will rename its Git branching structure to comply with TractusX release guidelines, and to be able to
+leverage
+GitHub convenience features, while continuing to use the Gitflow branching model.
+
+## Rationale
+
+The TractusX organization has established
```

Comment:
the proper writing of Tractus-X is Eclipse Tractus-X. (see the - between name)

The first usage of Eclipse Tractus-X should contain Eclipse while the next one can ommit it